js/react/utils/mantineProviderWrapper.jsx

```diff
@@ -1,7 +1,81 @@
 import { MantineProvider, createTheme } from '@mantine/core';
 
 const theme = createTheme({
-	/** Put your mantine theme override here */
```

Comment:
Are these the colors that we are already using?

Reply:
I actually have to check them again, I noticed that the mantine generator does not properly use the input colors, so I'll update them again